app/lib/actions/pulp3/repository/presenters/content_unit_presenter.rb

```diff
@@ -12,7 +12,7 @@ def progress
           def humanized_details
             ret = []
             ret << _("Cancelled.") if cancelled?
-            ret << _("Total tasks: ") + ": #{finished_units}/#{total_units}"
+            ret << _("Total tasks: ") + "#{finished_units}/#{total_units}"
```

Comment:
Mentioned this offline, but i think 'tasks' is even more confusing in this context, 'steps' or just 'Progress' might be better

Reply:
Reworded tasks to steps..